notebooks/xvcfmerge_array_input/main.py

```diff
@@ -26,113 +29,152 @@
     the tar.gz to your workspace using the utilities found in the
     [unarchive-tar-files-to-workspace notebook](https://app.terra.bio/terra.biodatacatalyst.nhlbi.nih.gov/#workspaces/biodata-catalyst/BioData%20Catalyst%20Collection/notebooks/launch/unarchive-tar-files-to-workspace.ipynb).
 
-    ## Workflows
+    ### Workflows
 
     Import the [Dockstore](https://dockstore.org) workflows into your workspace using the "NHLBI Biodata Catalyst" launch button:
      - [xvcfmerge](https://dockstore.org/workflows/github.com/DataBiosphere/xvcfmerge:v0.1.0?tab=info)
 
-    Workflow execution time is typically ~20 minutes per VCF.
+    Runtime and cost can vary greatly depending on the size and number of intput files. Sample runtime and cost
+    information is available for previous merge workflows in
+    [NHLBI BioData Catalyst / Utilities](https://dockstore.org/organizations/bdcatalyst/collections/Utilities) for
```

Comment:
is the "for" intended to be at the end of this sentence?

Reply:
I like to sprinkle in random prepositions to keep people on their toes.

will fix.